frontend/docs/testing.md

```diff
@@ -0,0 +1,362 @@
+# Frontend Testing Strategy & Plan
```

Comment:
@ghernandez345 @lukeheath Hey y'all!  This can move to  the "Contributing" section of the Fleet docs.  

I think if you move it to [this folder](https://github.com/fleetdm/fleet/tree/e96fb9df30dde80a65ae28c6c43482555db5fffb/docs/Contributing) and pattern match the `<meta>` tags from one of the other files in that folder, then everything should "just work", including nav.  

If you find after merging this that it is not the case, then just let Eric Shaw know and he can assist!

cc @chris-mcgillicuddy 

Context: https://github.com/fleetdm/confidential/issues/1530#issuecomment-1234678533